recipe/meta.yaml

```diff
@@ -41,13 +43,17 @@ test:
 
 about:
   home: https://github.com/python-trio/trustme
```

Comment:
This is not the home URL given by upstream. It should be: https://trustme.readthedocs.io/

Reply:
Fixed this